.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install Node.js
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: 14.4.0
+      - name: Install
+        run: npm install
+      - name: Lint
+        run: npm run lint
```

Comment:
I had serious issues getting yarn to work in another project, so I'm just using npm right now. Will need to re-visit this when we have time and figure out how to use yarn in workflows (and set up caching, etc)